src/test/java/org/testcontainers/containers/TarantoolContainerIT.java

```diff
@@ -73,31 +74,14 @@ public void testContainerWithParameters() throws Exception {
 
     @Test
     public void testContainerWithTrueVersion() throws Exception {
```

Comment:
In matrix tests env var TARANTOOL_VERSION always set. So this test is redundant.